.github/workflows/deploy-published-releases.yaml

```diff
@@ -27,32 +27,38 @@ jobs:
           npm ci
```

Comment:
Since the SDK is no longer a private package, we can remove this line and the `CROCT_BOT_TOKEN` secret.

Reply:
Sorry, its one line above this. Github bug apparently